devtools/common/src/main/filtered/extensions.json

```diff
@@ -219,6 +219,14 @@
     "artifactId": "quarkus-kubernetes",
     "guide": "https://quarkus.io/guides/kubernetes-guide"
   },
+  {
+    "name": "Kubernetes Client",
+    "labels": [
+      "kubernetes-client"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-kubernetes-client"
```

Comment:
Same here about the alphabetical order.

Reply:
I moved the AWS one since that was the one that looked out of place :)